src/Components/ErrorPage.tsx

```diff
@@ -0,0 +1,7 @@
+function ErrorPage() {
+  return (
+    <h1>Page Not Found 404</h1>
```

Comment:
```suggestion
    <h1 className="title">Page Not Found</h1>
```
![image](https://github.com/mate-academy/react_tabs-with-router/assets/96409563/e9ae30a4-be65-4794-ad7d-08033bff4c3e)